### DISCUSSION TITLE
Lua Page not visible

### DISCUSSION THREAD

**Discussion by 3D-Cat** [FrSkyRC/ETHOS-Feedback-Community#4836] [Q&A] [answered]
in Model -> second Page  Lua Page is not visible
TX is X20S, ETHOS 1.5.19 and Internal module FW is 2.2.7 (EU-LBT)
does anyone know the reason

![IMG_0449](https://github.com/user-attachments/assets/2b1377d0-e3ee-4606-8847-c40e64f912e5)
![IMG_0450](https://github.com/user-attachments/assets/ed5a592c-de29-4069-8b33-02781dd93975)

**Comment by bsongis** [comment]
Do you have a Lua script which adds something to this screen like a Lua Task or a Lua Source? Otherwise it's normal that it is not displayed, there will be nothing inside!

**Comment by 3D-Cat** [comment]
I tried it with a lua script 

![Screenshot 2024-12-04 135101](https://github.com/user-attachments/assets/6b6ce0c5-430c-44e0-adac-1c2cb509bfbf)

**Comment by bsongis** [comment]
But it is not a Task or a Source, yes?

**Comment by 3D-Cat** [comment]
it just shows the stick movement as simple graphic
is there any example in (ETHOS-Feedback-Community/lua/examples) I can try

**Comment by bsongis** [comment, marked as answer]
Yes there are a lot of examples and you know where they are it seems!
